src/include/fs_types.h

```diff
@@ -6,6 +6,38 @@
 #include "types.h"
 class JSONObj;
 
+#define CEPHFS_EBLOCKLISTED    EBLOCKLISTED
```

Comment:
There's no current value in these new definitions because they are based on the system values (exactly what we want to avoid!). Figure out what the Linux values are and define them here to the literal integer values.

Reply:
ack
this is fixed